lib/listener/mac/ITunesListener.cpp

```diff
@@ -228,19 +248,30 @@ ITunesListener::callback( CFDictionaryRef info )
 
 }
 
+QString //static
+ITunesListener::getPlayerAppId()
+{
+    const char* code = 
+    "try\n"
+        "tell me to get application id \"com.apple.Music\"\n"
+        "return \"com.apple.Music\"\n"
+    "on error\n"
+        "return \"com.apple.iTunes\"\n"
+    "end try\n";
+    return AppleScript( code ).exec();
```

Comment:
Does this work correctly when Music isn't running?

Reply:
Yes - the app doesn't need to be running for it to be found by this script.